odd-collector/pyproject.toml

```diff
@@ -55,6 +55,7 @@ couchbase = "^4.1.3"
 pyhive = { version = "^0.6.5", extras = ["hive"] }
 duckdb = "^0.8.0"
 opensearch-py = "2.3.2"
+pytest-aioresponses = "^0.2.0"
```

Comment:
Why do we use `pytest-aioresponses` and not just `aioresponses`? As I can see https://github.com/pheanex/pytest-aioresponses is kind of abandoned and I would like to avoid relying on that kind packages in long term. It can feel suitable at the moment but locks us in a lot of ways about maintainablility.

Reply:
It's better just copy this [pytest_aioresponses.py](https://github.com/pheanex/pytest-aioresponses/blob/main/pytest_aioresponses.py) and non drag the whole package with it's dependencies locks

Reply:
I installed this lib by mistake, actually didn't use that fixture. I removed pytest-aioresponses from dependencies.